variables.md

```diff
@@ -365,7 +365,7 @@ PUBLIC_CLOUD_LTP | boolean | false | If set, the run_ltp test module is added to
 PUBLIC_CLOUD_MAX_INSTANCES | integer | 1 | Allows the test to call "create_instance" subroutine within lib/publiccloud/provider.md a limited amount of times. If set to 0 or undef, it allows an unlimited amount of calls.
 PUBLIC_CLOUD_NAMESPACE | string | "" | The Public Cloud Namespace name that will be used to compose the full credentials URL together with `PUBLIC_CLOUD_CREDENTIALS_URL`.
 PUBLIC_CLOUD_NEW_INSTANCE_TYPE | string | "t3a.large" | Specify the new instance type to check bsc#1205002 in EC2
-PUBLIC_CLOUD_NO_CLEANUP | boolean | false | Do not remove the instance after test finished running.
+PUBLIC_CLOUD_NO_FINALACTIONS | boolean | false | Do not run the `publiccloud::basetest::finalActions()`.
```

Comment:
```suggestion
PUBLIC_CLOUD_NO_FINALACTIONS | boolean | false | Do not run the instance cleanup routines
```
A good documentation gives a human readable summary.
Pointing at code and then make the user go and read a whole routine is like "giving homework" and might upset the reader :wink:

Reply:
Now?